traffic_ops/traffic_ops_golang/servercapability/servercapability.go

```diff
@@ -131,17 +129,10 @@ func (v *TOServerCapability) Update(h http.Header) (error, error, int) {
 		return fmt.Errorf("cannot find exactly one server capability with the query string provided"), nil, http.StatusBadRequest
 	}
 
-	// check if the name was being updated by someone else
-	var existingLastUpdated *tc.TimeNoMod
-	q := `SELECT last_updated FROM server_capability WHERE name = $1`
-	if err := v.ReqInfo.Tx.QueryRow(q, v.Name).Scan(&existingLastUpdated); err != nil {
-		if err == sql.ErrNoRows {
-			return errors.New("server capability was not found"), nil, http.StatusNotFound
-		}
-		return nil, errors.New("server capability update: querying: " + err.Error()), http.StatusInternalServerError
-	}
-	if !api.IsUnmodified(h, existingLastUpdated.Time) {
-		return errors.New("the resource has been modified since the time specified by the request headers"), nil, http.StatusPreconditionFailed
+	// check if the name field was being updated by someone else
```

Comment:
As is the case with the next review comment, you can just change this comment to be `check if the entity was already updated by someone else`

Reply:
Changed it to a general comment.